cmd/vcfToBed/vcfToBed.go

```diff
@@ -7,21 +7,18 @@ import (
 	"github.com/vertgenlab/gonomics/fileio"
 	"github.com/vertgenlab/gonomics/vcf"
 	"log"
-	"strings"
 )
 
 func vcfToBed(infile string, outfile string, delimiter string) {
 	ch, _ := vcf.GoReadToChan(infile)
-	var note string
+	var note string = ""
 	var err error
 	out := fileio.EasyCreate(outfile)
 	defer out.Close()
 
 	for v := range ch {
-		words := strings.Split(v.Notes, "\t")
-		note = words[0]
-		for i := 0; i < len(words); i++ {
-			note = note + delimiter + words[i]
+		for i := 0; i < len(v.Samples); i++ {
+			note = note + delimiter + vcf.HelperSamplesToString(v.Samples, i)
```

Comment:
Maybe SamplesToString?  A helper function would usually be a non-public function that is called by a more user-friendly function, often as the recursive step.
Does "note" need to get reset before entering this loop?  Otherwise it seems like "note" would get bigger and bigger as it goes over all the values coming out of the channel